akeneo-design-system/src/components/Dropdown/Item/Item.tsx

```diff
@@ -119,9 +139,10 @@ const Item = React.forwardRef<HTMLDivElement, ItemProps>(
     return (
       <ItemContainer
         tall={tall}
-        tabIndex={actionableRef.current === null ? 0 : -1}
+        tabIndex={null === actionableRef.current ? 0 : -1}
```

Comment:
To disable the keyboard navigation

```suggestion
        tabIndex={null === actionableRef.current && !disabled ? 0 : -1}
```

Reply:
I tried at first but it make the following Item unreachable with the arrow navigation, so I decided to leave it navigable but not actionable